### ISSUE TITLE
Alternatives

### ISSUE THREAD

**Issue by stalniy** [pedsmoreira/allowed#1]
Hi there

I’m author of CASL (https://github.com/stalniy/casl) which is quite similar to what you want to do. 

Also there are other libraries which were built based on Ruby cancan - https://npms.io/search?q=cancan

So, maybe it would better contribute to existing libs instead of creating a new one? :)

Anyway that is a good initiative!

**Comment by pedsmoreira** [comment]
Hey @stalniy,

Thanks for reaching out. I just came across this issue and your library seems fantastic!

I found this [cancan](https://github.com/vadimdemedes/cancan) implementation a few months but the API wasn't quite what I needed for my side project - [GitShowcase](https://www.gitshowcase.com/).

Your library's API seem to be exactly what I was looking for, so I'll use it :).

I'll reach out to NPM and see if they can have the `allowed` name freed, in case someone else wishes to use it.
